### PR TITLE
Added support for bing.com

### PR DESCRIPTION
**Pull request by xme**
Hi Chris,

For my day 2 day job, I've to search for websites hosted behind a specific IP. I added a module to extract hostnames using the "IP=<ip>" query in Bing. The user needs a Bing API key.

Update:
I also added a feature to pass a filename to 'save' and 'export' commands. It's easier for me as I'm running JustMetaData in a Docker container and need to store results in a volume for persistence.

/x

### CODE REVIEW

**Comment by ChrisTruncer** [comment]
This looks awesome!  Thanks for sending this over, I'll get checking it out.

File: common/orchestra.py [hunk 1]
```diff
@@ -103,24 +103,32 @@ def collapse(self, var, tabs=0):
...
-
-        print helpers.color("\nExport file saved to disk at export_" + current_date + "_" + current_time + ".csv")
+            try:
+                with open(f, 'a') as export_file:
+                    csv_file = csv.DictWriter(export_file, attrs.keys())
+                    if add_header:
+                        csv_file.writeheader()
+                        add_header = False
+                    csv_file.writerow(attrs)
+                    print helpers.color("\nExport file saved to disk at " + f)
```
Comment (ChrisTruncer) [inline common/orchestra.py]: Can you move this print statement out of the for loop?  Right now it prints to the console for each IP loaded into Just-Metadata.

**Review by ChrisTruncer — CHANGES_REQUESTED** [review]
I only have the one small request for moving the print statement out of the for loop.

Thanks for getting this sent over.

**Comment by xme** [comment]
Stupid me :) Fixed.

**Review by ChrisTruncer — APPROVED** [review]

**Comment by ChrisTruncer** [comment]
Merging this in, thanks for the contribution!

**Pull request merged**